ERCS/erc-7660.md

```diff
@@ -0,0 +1,426 @@
+---
+eip: 7660
+title: Release tokens on predefined periods
+description: Controlled by smart contracts, tokens can be linearly released based on the predefined period.
+author: Ali Kaya (@AliKaya) <ali.kayatur747@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/erc-7660-a-protocol-for-erc20-tokens-which-released-based-on-predefined-periods-controlled/19305
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-08
+requires: 20
+---
+
+## Abstract
+
+This proposal outlines [ERC-20](./eip-20.md) tokens which released based on predefined periods controlled by smart contracts,smart contract allows token released based on predefined periods, enabling lock token balances and gradually release tokens.
```

Comment:
Your abstract is light on technical details. You should describe, at a high level, how your proposal accomplishes its goals.